bin/node_init

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# LEAP Platform node initialization.
+# This script is run on the target server when `leap node init` is run.
+#
+
+DEBIAN_VERSION="^(jessie|8\.)"
+LEAP_DIR="/srv/leap"
+HIERA_DIR="/etc/leap"
+INIT_FILE="/srv/leap/initialized"
+REQUIRED_PACKAGES="puppet rsync lsb-release locales"
+
+APT_GET="apt-get -q -y -o DPkg::Options::=--force-confold"
+APT_GET_UPDATE="apt-get update -o Acquire::Languages=none"
+BAD_APT_RESPONSE="(BADSIG|NO_PUBKEY|KEYEXPIRED|REVKEYSIG|NODATA|Could not resolve|failed to fetch)"
+export DEBIAN_FRONTEND=noninteractive
+
+test -f $INIT_FILE && rm $INIT_FILE
+if ! egrep -q "$DEBIAN_VERSION" /etc/debian_version; then
+  echo "ERROR: This operating system is not supported. The file /etc/debian_version must match /$DEBIAN_VERSION/ but is: `cat /etc/debian_version`"
+  exit 1
+fi
+mkdir -p $LEAP_DIR
+echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
+
+#
+# UPDATE PACKAGES
+# (exit code is not reliable, sadly)
+#
+echo "updating package list"
+
+error_count=0
+while read line; do
+  error=$(echo $line | egrep "$BAD_APT_RESPONSE")
+  if [[ $error ]]; then
+    errors[error_count]=$error
+    ((error_count++))
+    break # should we halt on first error?
+  fi
+  echo $line
+done < <($APT_GET_UPDATE 2>&1)
+
+if [[ $error_count > 0 ]]; then
+  echo "ERROR: fatal error in 'apt-get update', bailing out."
+  for e in "${errors[@]}"; do
+    echo "       $e"
+  done
+  exit 1
+fi
+
+#
+# UPDATE TIME
+#
+echo "updating server time"
+test -f /etc/init.d/ntp && /etc/init.d/ntp stop
+test -f /usr/sbin/ntpdate || $APT_GET install ntpdate
+echo "running ntpdate..."
+test -f /usr/sbin/ntpdate && ntpdate 0.debian.pool.ntp.org 1.debian.pool.ntp.org 2.debian.pool.ntp.org 3.debian.pool.ntp.org
```

Comment:
Remove lines 56-58 and instead put this:

/usr/sbin/ntpd -gxq >/dev/null

if you want to capture the output for the log, then omit the >/dev/null, the output looks like this:
<pre>
root@muck:/home/micah# /usr/sbin/ntpd -gxq
16 Feb 11:58:51 ntpd[4395]: ntpd 4.2.8p4@1.3265-o Tue Nov  3 02:26:54 UTC 2015 (1): Starting
16 Feb 11:58:51 ntpd[4395]: Command line: /usr/sbin/ntpd -gxq
16 Feb 11:58:51 ntpd[4395]: proto: precision = 0.039 usec (-25)
16 Feb 11:58:51 ntpd[4395]: Listen and drop on 0 v6wildcard [::]:123
16 Feb 11:58:51 ntpd[4395]: Listen and drop on 1 v4wildcard 0.0.0.0:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 2 lo 127.0.0.1:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 3 eth0 192.168.1.108:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 4 tun0 10.42.0.13:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 5 lo [::1]:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 6 eth0 [fdd4:d95:f687:0:d250:99ff:fe43:38d0]:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 7 eth0 [fd37:c49:9e32:0:d250:99ff:fe43:38d0]:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 8 eth0 [fe80::d250:99ff:fe43:38d0%2]:123
16 Feb 11:58:51 ntpd[4395]: Listen normally on 9 tun0 [2001:db8:123::100b]:123
16 Feb 11:58:51 ntpd[4395]: Listening on routing socket on fd #26 for interface updates
16 Feb 11:58:52 ntpd[4395]: Soliciting pool server 199.204.45.235
16 Feb 11:58:53 ntpd[4395]: Soliciting pool server 142.137.247.109
16 Feb 11:58:53 ntpd[4395]: Soliciting pool server 192.95.20.208
16 Feb 11:58:54 ntpd[4395]: Soliciting pool server 208.80.96.96
16 Feb 11:58:54 ntpd[4395]: Soliciting pool server 207.210.46.249
16 Feb 11:58:54 ntpd[4395]: Soliciting pool server 2001:470:e838:1:123::2
16 Feb 11:58:54 ntpd[4395]: ntpd: time slew -0.003369 s
ntpd: time slew -0.003369s
root@muck:/home/micah# 
</pre>